cmd/osv-scanner/main.go

```diff
@@ -147,13 +157,26 @@ func run(args []string, stdout, stderr io.Writer) int {
 				return fmt.Errorf("failed to write output: %w", errPrint)
 			}
 
+			if outputPath != "" {
+				file, err := os.Create(outputPath)
```

Comment:
Can we just use `https://pkg.go.dev/os#WriteFile` here?

Reply:
Ah this is circumvented by resolving this: https://github.com/google/osv-scanner/pull/420#discussion_r1244619219